src/tests/jobtest.c

```diff
@@ -30,26 +30,131 @@
 
 #include <stdio.h>
 #include <string.h>
+#include <strings.h>
 #include <daos.h>
+#include <daos_mgmt.h>
+#include <daos/common.h>
+#include <daos/pool.h>
+
+static char *progname;
+
+void print_usage(void)
+{
+	fprintf(stderr, "Usage: %s -p pool_str [-s nsecs] [-x] "
+			"[-h handles_per_pool]\n", progname);
+}
+
+void cleanup_handles(uuid_t *pool_uuids, int num_pools,
+		     daos_handle_t **pool_handles, int handles_per_pool)
+{
+	int i, rc;
+
+	for (i = 0; i < num_pools; i++) {
+		int idx;
+
+		if (uuid_is_null(pool_uuids[i])) {
+			continue;
+		}
+		for (idx = 0; idx < handles_per_pool; idx++) {
```

Comment:
very minor nit, but for nested loops, sometimes it's nice to stick with i, j, k, etc. i vs. idx is slightly confusing.

Reply:
Ok fixed.